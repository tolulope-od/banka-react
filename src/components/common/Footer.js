import React from 'react';

const Footer = () => {
  return (
    <footer className="footer">
      <p className="footerTxt center-text">Banka Inc &copy; 2019</p>
    </footer>
  );
};

export default Footer;
