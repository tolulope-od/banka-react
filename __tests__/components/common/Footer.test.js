import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../../src/components/common/Footer';

describe('<Footer />', () => {
  it('Renders the footer component', () => {
    const wrapper = shallow(<Footer />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
