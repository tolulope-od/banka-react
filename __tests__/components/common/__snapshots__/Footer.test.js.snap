// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Renders the footer component 1`] = `
<footer
  className="footer"
>
  <p
    className="footerTxt center-text"
  >
    Banka Inc Â© 2019
  </p>
</footer>
`;
